tests/unit-tests/test-class-utils.php

```diff
@@ -519,4 +519,77 @@ public function testGetTargetResumeId_WhenCalled_ReturnsCourseIdIfItHasNoLessons
 		/* Assert */
 		$this->assertEquals( $course_id, $result );
 	}
+
+	public function testIsFseThemeSupported_WhenCalled_WorksIfBlockTemplatesIndexAvailable() {
+		/* Arrange */
+		$theme_directory = get_template_directory() . '/block-templates';
+		$index_file      = $theme_directory . '/index.html';
+
+		// Remove the 'block-templates/index.html' file if it exists.
+		if ( file_exists( $index_file ) ) {
+			unlink( $index_file );
+		}
+
+		// Create the 'block-templates' directory if it doesn't exist.
+		if ( ! is_dir( $theme_directory ) ) {
+			mkdir( $theme_directory );
+		}
+
+		$this->create_file( $index_file );
+
+		/* Assert */
+		// Call the function and assert the result.
+		$this->assertTrue( Sensei_Utils::is_fse_theme() );
+
+		unlink( $index_file );
+		rmdir( $theme_directory );
+	}
+
+	public function testIsFseThemeSupported_WhenCalled_WorksIfIndexHtmlAvailable() {
+		/* Arrange */
+		$theme_directory = get_template_directory() . '/templates';
+		$index_file      = $theme_directory . '/index.html';
+
+		// Remove the 'templates/index.html' file if it exists.
+		if ( file_exists( $index_file ) ) {
+			unlink( $index_file );
+		}
+
+		// Create the 'templates' directory if it doesn't exist.
+		if ( ! is_dir( $theme_directory ) ) {
+			mkdir( $theme_directory );
+		}
+
+		$this->create_file( $index_file );
+
+		/* Assert */
```

Comment:
Maybe `/* Act & Assert */`?

My own preference is to split these two sections to make clear what is the main action of the test (what exactly we test) and what outcome we verify.